flake.nix

```diff
@@ -129,13 +144,22 @@
 
           src = ./site/.;
           # Required for the `canvas` package!
-          extraBuildInputs = with pkgs; [
-            cairo
-            pango
-            pixman
-            libpng libjpeg giflib librsvg
-            python312Packages.setuptools
-          ] ++ ( lib.optionals stdenv.targetPlatform.isDarwin [ darwin.apple_sdk.frameworks.Foundation xcbuild ] );
+          extraBuildInputs =
+            with pkgs;
+            [
+              cairo
+              pango
+              pixman
+              libpng
+              libjpeg
+              giflib
+              librsvg
+              python312Packages.setuptools
+            ]
+            ++ (lib.optionals stdenv.targetPlatform.isDarwin [
+              darwin.apple_sdk.frameworks.Foundation
+              xcbuild
+            ]);
           installInPlace = true;
```

Comment:
Are you using a formatter like https://github.com/kamadorueda/alejandra?

Reply:
Yes. [`nixfmt-rfc-style`](https://github.com/NixOS/nixfmt) to be more specific.
Its also defined in line 44 and included in the devshell.

Reply:
How are we keeping this file up to date? Is it manual, i.e., whenever someone updates the Nix flake?

Reply:
Yes.
Thank you.